temba/orgs/migrations/0074_remove_topup_expires_on.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.10 on 2021-06-28 13:33
+
+from django.db import migrations, models
+from django.utils.timezone import now as tz_now, timedelta
+
+
+def year_from_now():
+    return tz_now() + timedelta(days=365)
```

Comment:
Let's make a migration to update the expiration time for something like 50 years from now